test/buildpacks_registry_api_test/cache_client_test.exs

```diff
@@ -0,0 +1,151 @@
+defmodule BuildpacksRegistryApi.CacheClientTest do
+  alias BuildpacksRegistryApi.CacheClient
+  use ExUnit.Case
```

Comment:
I think these could be async as we don't test the cache client elsewhere? (Async works per file not per test)

Reply:
Yep I took that out when debugging something and forgot to put it back in. Good catch 👍🏻